### ISSUE TITLE
Endgun Configuration needs clarifications

### ISSUE THREAD

**Issue by knelson-farmbeltnorth** [ADAPT/Standard#15]
Within Endgun Table Entry are all values optional?   Perhaps throw value is required?

How can the Endgun Configuration have either 1 or multiple table entries.    A table entry makes sense in and of itself, but if there are multiples, what is the differentiation in scope of the multiples?

If an Endgun table is just a collection of Endgun Table Entries, why do we need an Endgun table?   We can just have the collection of entries.

What is the distinction between members of the Endgun table?  There is no logical "primary key," just a collection of values without any scoping information on the values.

**Comment by knelson-farmbeltnorth** [comment]
Following detailed discussion with @aferreyra, we agreed to rename Endgun Table Entry to Endgun Configuration Record.

Taking the above questions in reverse: 

The logical primary key is Pressure.
We don't need the Endgun table.   We have removed it.
The one vs. multiple allows for setting a single default set of values based on one pressure or a more detailed configuration based on multiple pressures.    Comments will indicate that only the nominal/default or the collection may be set.
We've made all 3 values of the Endgun Configuration Record as required at this time, following cardinality in PAIL.
